source/jsonnet/northern-ireland/household/blocks/who-lives-here/any_more_people_living_here.jsonnet

```diff
@@ -22,7 +21,7 @@ local questionTitle = {
 local summaryTitlePersonName = {
   text: '{person_name}',
   placeholders: [
-    placeholders.personName,
+    placeholders.personName('list_has_same_name_items'),
```

Comment:
All occurrences of `personName('list_has_same_name_items')` should now be `personName(includeMiddleNames='if_same_names_exist')`.